test/commands/lightning/dev/site.test.ts

```diff
@@ -4,39 +4,62 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+
+import { strictEqual } from 'node:assert';
 import { TestContext } from '@salesforce/core/testSetup';
-// import { expect } from 'chai';
-// import { stubSfCommandUx } from '@salesforce/sf-plugins-core';
-// import LightningDevSite from '../../../../src/commands/lightning/dev/site.js';
+import { expect } from 'chai';
+import { Org } from '@salesforce/core';
+import { LocalDevOptions } from '@lwrjs/api';
+import LightningDevSite from '../../../../src/commands/lightning/dev/site.js';
+import { OrgUtils } from '../../../../src/shared/orgUtils.js';
 
-// TODO fix me once we have a fully working command
 describe('lightning dev site', () => {
   const $$ = new TestContext();
-  // let sfCommandStubs: ReturnType<typeof stubSfCommandUx>;
 
   beforeEach(() => {
-    // sfCommandStubs = stubSfCommandUx($$.SANDBOX);
+    $$.SANDBOX.stub(OrgUtils, 'isLocalDevEnabled').resolves(true);
+    $$.SANDBOX.stub(OrgUtils, 'ensureMatchingAPIVersion').returns();
   });
 
   afterEach(() => {
     $$.restore();
   });
 
-  it('runs hello', async () => {
-    // await LightningDevSite.run([]);
-    // const output = sfCommandStubs.log
-    //   .getCalls()
-    //   .flatMap((c) => c.args)
-    //   .join('\n');
-    // expect(output).to.include('hello world');
+  it('should have summary, description, and examples defined', () => {
+    strictEqual(typeof LightningDevSite.summary, 'string', 'Summary should be a string');
+    strictEqual(typeof LightningDevSite.description, 'string', 'Description should be a string');
+    strictEqual(typeof LightningDevSite.examples, 'object', 'Examples should be an array');
+  });
+
+  it('result should be undefined if local development is not enabled', async () => {
```

Comment:
We probably should do something similar to LWC here:
![image](https://github.com/user-attachments/assets/00b534a8-a69e-4dc2-b83c-708798bcb5ae)

Reply:
if you look at the beforeEach method, they are mocking org data, auth, etc. Thats what you need to get the unit tests running:

```
beforeEach(async () => {
    stubUx($$.SANDBOX);
    stubSpinner($$.SANDBOX);
    await $$.stubAuths(testOrgData);

    $$.SANDBOX.stub(SfConfig, 'create').withArgs($$.SANDBOX.match.any).resolves(SfConfig.prototype);
    $$.SANDBOX.stub(SfConfig, 'addAllowedProperties').withArgs($$.SANDBOX.match.any);
    $$.SANDBOX.stub(SfConfig.prototype, 'get').returns(undefined);
    $$.SANDBOX.stub(SfConfig.prototype, 'set');
    $$.SANDBOX.stub(SfConfig.prototype, 'write').resolves();
    $$.SANDBOX.stub(Connection.prototype, 'getUsername').returns(testUsername);
    $$.SANDBOX.stub(PreviewUtils, 'getOrCreateAppServerIdentity').resolves(testIdentityData);
    $$.SANDBOX.stub(OrgUtils, 'isLocalDevEnabled').resolves(true);
    $$.SANDBOX.stub(OrgUtils, 'ensureMatchingAPIVersion').returns();

    MockedLightningPreviewApp = await esmock<typeof LightningDevApp>('../../../../src/commands/lightning/dev/app.js', {
      '../../../../src/lwc-dev-server/index.js': {
        startLWCServer: async () => ({ stopServer: () => {} }),
      },
    });
  });
```